MANIFEST.in

```diff
@@ -1,5 +1,5 @@
-recursive-include material *.js *.css *.html *.svg *.png *.yml
-recursive-include material *.ttf *.woff *.woff2
+recursive-include brudi *.js *.css *.html *.svg *.png *.yml
```

Comment:
note: this is the actual change: include `brudi` folder for a valid mkdocs theme package on https://pypi.org/project/mkdocs-brudi